testing/file_test/file_test_base.h

```diff
@@ -148,12 +132,29 @@ class FileTestBase : public testing::Test {
   static auto TransformExpectation(int line_index, llvm::StringRef in)
       -> testing::Matcher<std::string>;
 
-  const std::filesystem::path* path_;
+  const std::filesystem::path path_;
+};
+
```

Comment:
Maybe a bit of documentation would be useful here?

```suggestion
// A plain struct to aggregate a name and factory function for tests using this framework.
```